packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -240,7 +242,7 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
               isMonthPickerVisible={ this.props.isMonthPickerVisible }
               showMonthPickerAsOverlay={ this.props.showMonthPickerAsOverlay }
               today={ this.props.today }
-              value={ selectedDate }
+              value={ selectedDate || initialPickerDate }
```

Comment:
>initialPickerDate [](start = 38, length = 17)

Please make sure that the default behavior is not changing for existing consumers after this change